### PR TITLE
Adjust README and meta and move examples

### PR DESCRIPTION
**Pull request by wawrzek**
- example doesn't work with new version so move it aside
- adjust description to match recent changes to to make clear that
  example is old

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,29 +51,24 @@ All variables are defined in *defaults/main.yml*. Some of them should be changed
...
-
-This is a summary of the actions performed by Ansible:
-
-- Install necessary packages (including OpenJDK from zulu.org by default)
-- Create a user and a directory for Corda
-- Create the systemd configuration for Corda node
-- Prepare Corda configuration file (*node.conf*)
-- Install Corda jar (node) either from Maven Central, or from your local machine.
-- Optional node can be register with Doorman
+## Old Example (priori to version 3)
```
Comment (przemolb) [inline README.md]: prior

File: README.md [hunk 2]
```diff
@@ -1,13 +1,18 @@
...
-- Modify the **hosts** file and fill it with valid information about *city* (just for location on a node on network map), *legal node name* (for *network map*), *email address* and node *role*. This information is going to be used by Ansible to create a valid *node.conf* file.
-- run `ansible-playbook -i hosts corda.yml`
+This is a summary of the actions performed by Ansible:
+
+- Install necessary packages (including OpenJDK from zulu.org by default)
+- Create a user and a directory for Corda
+- Create the systemd configuration for Corda node
+- Prepare Corda configuration file (*node.conf*)
+- Install Corda jar (node) either from Maven Central, from your local machine (e.g. to use with Corda Enterprise) or snapshot from R3 artifactory
+- Optional node can be register with Doorman
```
Comment (przemolb) [inline README.md:15]: Optionally ? Can it work without doorman ?

File: README.md (hunk 1 above)
Comment (wzur-r3) [inline README.md]: `priori` is a typo, I believe

**Review by wzur-r3 — COMMENTED** [review]
Frankly speaking I would prefer to have a Vagrantfile directly in the root of this project, being a part of the actual development process of this role.

Otherwise I'm fine with this PR.

**Review by przemolb — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (wawrzek) [reply README.md:15]: In theory you could bind this role with Bootstrapper.

**Pull request merged**